features/dd-sdk-android-rum/src/test/kotlin/com/datadog/android/rum/internal/domain/event/RumEventSerializerTest.kt

```diff
@@ -748,6 +750,110 @@ internal class RumEventSerializerTest {
         }
     }
 
+    @RepeatedTest(8)
```

Comment:
we should use `@Test` to reduce CI load after making repeating test in local.

Reply:
in this case we need a repeated test as this model has multiple variables variation. Is good to have repetitive tests in order to make sure we cover more cases in this variation.